plans/rolling.pp

```diff
@@ -131,6 +131,12 @@
     fail_plan("Unable to update the target branch ${target_branch} to SHA ${sha}")
   }
 
+  $code_result = cd4pe_deployments::deploy_code($target_environment)
+  $validate_code_deploy_result = cd4pe_deployments::validate_code_deploy_status($code_result)
+  unless ($validate_code_deploy_result[error] =~ Undef) {
```

Comment:
Looks like most error checking in this file doesn't do the `=~ Undef`.

I have no idea what's right or wrong, just pointing out the difference in case it's meaningful.

Reply:
There's another usage of this in the plan so I think this is :+1:  Also, I think it's better to use type matching like this instead of relying on the variable to be truthy.